types/jest/index.d.ts

```diff
@@ -29,7 +29,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.0
 
-/// <reference types="jest-diff" />
+/// <reference types="expect" />
```

Comment:
```suggestion
import { MatcherState as JestMatcherState } from "expect";
```

Reply:
That won't work, unfortunately. See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/32891#discussion_r255284875

Would be lovely to fix it, though.

(We don't need the `references` I'll rebase this after #41852 is merged)

Reply:
In&nbsp;that&nbsp;case you&nbsp;can do&nbsp;<…>.

Reply:
In&nbsp;that&nbsp;case you&nbsp;can do&nbsp;<…>.

Reply:
yeah, good call. done 🙂